From 11b3186b8b8d93c8a8a583e3a9640f4c18de6cc8 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Tue, 27 Oct 2020 09:35:33 +0100
Subject: [PATCH] [tests] fix hyq paths

`loadRobotModel` (through private `makeeModelPath`) assumes that the
subdir is the same as the extension, so use loadModel instead.

ref https://github.com/humanoid-path-planner/hpp-pinocchio/issues/134
---
 tests/tools-fullbody.hh | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/tests/tools-fullbody.hh b/tests/tools-fullbody.hh
index 3569ea9..7b63efc 100644
--- tests/tools-fullbody.hh
+++ tests/tools-fullbody.hh
@@ -305,15 +305,12 @@ hpp::pinocchio::RbPrmDevicePtr_t loadTalosLEGAbsract() {
 RbPrmFullBodyPtr_t loadHyQ() {
   const std::string robotName("hyq");
   const std::string rootJointType("freeflyer");
-  const std::string packageName("example-robot-data/robots/hyq_description");
-  const std::string modelName("hyq");
-  const std::string urdfSuffix("_no_sensors");
-  const std::string srdfSuffix("");
 
   hpp::pinocchio::DevicePtr_t device = hpp::pinocchio::Device::create(robotName);
-  // hpp::pinocchio:: (device,rootJointType, packageName, modelName, urdfSuffix,srdfSuffix);
 
-  hpp::pinocchio::urdf::loadRobotModel(device, rootJointType, packageName, modelName, urdfSuffix, srdfSuffix);
+  hpp::pinocchio::urdf::loadModel(device, 0, "", rootJointType,
+                                  "package://example-robot-data/robots/hyq_description/robots/hyq_no_sensors.urdf",
+                                  "package://example-robot-data/robots/hyq_description/srdf/hyq.srdf");
   // [-2,5, -1, 1, 0.3, 4]
 
   /*DevicePtr_t robot =
